### ISSUE TITLE
Can't add a block at the end of a group

### ISSUE THREAD

**Issue by smerriman** [WordPress/gutenberg#18922]
Apologies if this has been reported; seems so obvious it must be somewhere, but have tried multiple searches and not found anything.

If I create a new block, a plus button will appear allowing me to add the first block to the group.

However, there is then no way to add another block to the end of the group. The plus button below the group adds it outside of the group; I can only insert a block before an existing one. (Of course, I can use the dots icon and 'insert after', but these seems unnecessarily complex).

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version 70.01

**Additional context**
Standard WordPress  5.3 installation with default TwentyTwenty theme

**Comment by talldan** [comment]
@smerriman Yes, this seems like a bug. Thanks for reporting it!

**Comment by corentin-gautier** [comment]
The plus button is missing after the last block either in a group or not, which is very unintuitive for the end user, I think the hint "Start writing or type / to choose a block" with the plus button should always be visible after the last block making it easy to add a block and understand where you can add blocks

**Comment by smerriman** [comment]
Is there any program on this? As per corentin-gautier's comment above, this isn't just affecting groups, but basically anywhere else inner blocks are used - eg the bottom of a column, or Media & Text block, etc.

I can see there's a hidden block appender div, but it's empty, and never filled with anything.

This seems a rather major bug considering the end of a particular section is the most likely place you're going to want to add new content?

**Comment by talldan** [comment]
@smerriman Yep, as far as I know the idea is that if the last block is a paragraph, typing is the easiest way to insert the next block. If it's an image or some other block there should be a '+' button afterwards.

In testing, this issue now seems to have been fixed, here's a screenshot showing the trailing appender that's now visible again in a group:
<img width="656" alt="Screenshot 2020-09-08 at 12 30 47 pm" src="https://user-images.githubusercontent.com/677833/92433286-32bd1c00-f1cf-11ea-96ce-29e52ab3e58b.png">

It seems like the group or one of the blocks within the group has to be selected for it to be present.
